pypeit/specobj.py

```diff
@@ -486,6 +486,7 @@ def update_flex_shift(self, shift, flex_type='local'):
         self.FLEX_SHIFT_TOTAL += shift
 
     # TODO This should be a wrapper calling a core algorithm.
+    #      It is, right? Can this TODO be removed?
```

Comment:
Yeah, I agree.  This effectively wraps `flux_calib.get_sensfunc_factor`, so I think you can remove the TODO.